python_modules/dagster/dagster/core/execution_plan/intermediates_manager.py

```diff
@@ -85,8 +86,8 @@ def has_value(self, step_output_handle):
 # will likely be to inject a different type of execution step that threads
 # the manager
```

Comment:
Do we still think this is true?

Reply:
some species of this is still true but you're right that the comment is now spurious